src/Be.Vlaanderen.Basisregisters.Aws.DistributedMutex/DynamoDBMutex.cs

```diff
@@ -220,5 +222,25 @@ public async Task<bool> DoesTableExists()
 
             return false;
         }
+
+        public async Task<bool> IsActiveAsync(string resourceId, CancellationToken cancellationToken = default)
+        {
+            var table = await GetTableAsync();
+            var doc = new Document
+            {
+                [ColumnNames.HolderId] = _id,
+                [ColumnNames.ResourceId] = resourceId
+            };
+
+            var response = await table.GetItemAsync(doc, new GetItemOperationConfig
+            {
+                AttributesToGet = new List<string> { ColumnNames.LeaseExpiry },
+                ConsistentRead = true
+            }, cancellationToken);
+
+            return response is not null
+                && DateTimeOffset.TryParse(response[ColumnNames.LeaseExpiry], out var dateTimeOffset)
```

Comment:
maybe rename `dateTimeOffset` to `lockExpirationDateTime` or something like that.

Reply:
fixed